components/threading/channel_threads/thread_footer/thread_footer.tsx

```diff
@@ -17,15 +17,14 @@ import SimpleTooltip from 'components/widgets/simple_tooltip';
 import {THREADING_TIME} from '../../common/options';
 
 type Props = {
-    participants: ComponentProps<typeof Avatars>['users'];
+    participants: Array<{id: string}>; // Post['participants']
     totalParticipants?: number;
```

Comment:
this now seems redundant, right? we can probably just get it from the length of `participants`

Reply:
For the purposes of the Avatars tooltip, more than (some threshold) may be too many to display—like with reactions tooltips, I think the first XX are named with the rest as numbered overflow ("and 23 others..."). In the GA/polish phase `participants` will be optimized around this, but for now just shows all.